Remove `initializationPrior` and rename `objectivePrior` (#612)
PEtab-dev/PEtab: doc/v2/documentation_data_format.rst

  1 or 0, depending on, if the parameter is estimated (1) or set to a fixed
  value(0) (see ``nominalValue``).

- ``prior`` [STRING, OPTIONAL]

>   Prior types used for the objective function during estimation or inference.

Reviewer: ```suggestion
  Prior types used for used for the :ref:`MAP objective function and for Bayesian inference <_v2_objective_function>`.
```

>   Prior types used for used for the :ref:`MAP objective function and for Bayesian inference <_v2_objective_function>`.

Reviewer: ```suggestion
  Prior types used for used for the :ref:`MAP objective function and for Bayesian inference <v2_objective_function>`.
```

>   If priors are specified for a subset of parameters, defaults to 

Reviewer: ```suggestion
  If priors are specified for a subset of parameters, this defaults to 
```

>   Prior types used for used for the :ref:`MAP objective function and for Bayesian inference <v2_objective_function>`.
>   If priors are specified for a subset of parameters, this defaults to 
>   ``parameterScaleUniform`` for any unspecified parameter.

Reviewer 2: ```suggestion
  Prior types used for the :ref:`MAP objective function and for Bayesian inference <v2_objective_function>`.
  It is valid to have no priors. However, if priors are specified for a subset
  of parameters, this defaults to the ``uniform`` prior (with ``priorParameters``
  set to ``lowerBound;upperBound``) for the other parameters.
```

  1 or 0, depending on, if the parameter is estimated (1) or set to a fixed
  value(0) (see ``nominalValue``).

- ``prior`` [STRING, OPTIONAL]

>   Prior types used for the :ref:`MAP objective function and for Bayesian inference <v2_objective_function>`.

Reviewer 3: state that including/not-including decides whether MLE/MAP is used as objective function?

Author: I think it is sufficient that this information is provided in the referenced section

    - *parameterScaleNormal*: Gaussian prior on original parameter scale
    - *parameterScaleLaplace*: Laplace prior on original parameter scale

- ``priorParameters`` [STRING, OPTIONAL]

>   Prior parameters used for the objective function during estimation.

Reviewer: ```suggestion
  Prior parameters used for the :ref:`MAP objective function and for Bayesian inference <_v2_objective_function>`.
```

>   ``priorParameters`` is required if ``prior`` is non-empty. If ``prior`` is specified for any other parameter, and ``prior`` is  empty for the given parameter, this defaults to ``lowerBound;upperBound``.

Reviewer 2: Moved to the `prior` description, since it seems clearer to me that way.
```suggestion
  ``priorParameters`` is required if ``prior`` is non-empty.
```
